app/src/main/res/values/strings.xml

```diff
@@ -679,7 +679,7 @@
     <string name="help_voice_record_title">Record Dramatically</string>
     <string name="help_voice_record_body">
         <![CDATA[Re-record the translation with a clear voice and dramatic expression.
-        <br/>(Then tap the pause button to pause recording, or tap the square to stop recording.)]]>
+        <br/>(Tap the pause button to pause recording, or tap the square to stop recording.)]]>
```

Comment:
Actually there shouldn't be any parenthesis here according to the XML file containing the changes